cmd/jujud/updateseries/updateseries_test.go

```diff
@@ -358,7 +300,7 @@ func (s *updateSeriesCmdSuite) TestRunPostUpstartToSystemdUpgradeReboot(c *gc.C)
 		"wrote jujud-unit-.*-0 agent, enabled and linked by systemd\n"+
 		"wrote jujud-unit-.*-0 agent, enabled and linked by systemd\n"+
 		"wrote jujud-machine-0 agent, enabled and linked by systemd\n"+
-		"successfully copied tools and relinked agent tools\n")
+		"successfully copied agent binaries and relinked binaries\n")
```

Comment:
I think this should be "successfully copied binaries and relinked agent binaries". It copies the binaries into a common location and then creates a link for each agent, so that would make it clearer.